CRM/ACL/Form/ACL.php

```diff
@@ -160,6 +160,7 @@ public function buildQuickForm() {
     $this->add('select', 'event_id', ts('Event'), $event);
 
     $this->add('checkbox', 'is_active', ts('Enabled?'));
+    $this->add('advcheckbox', 'deny', ts('Negate access'));
```

Comment:
@seamuslee001 this might make more sense as radios or a dropdown select with the label "Mode" and "Allow" and "Deny" as options. I've also suggested updating the table to match.